internal/controller/hierarchy.go

```diff
@@ -326,3 +326,53 @@ func GetDeployment(ctx context.Context, c client.Client, obj client.Object) (*ch
 		objWithName(&choreov1.DeploymentTrack{}, GetDeploymentTrackName(obj)),
 	)
 }
+
+func GetDeployableArtifact(ctx context.Context, c client.Client, obj client.Object) (*choreov1.DeployableArtifact, error) {
```

Comment:
Shall we move this to the controller as these resources don't have any child resources yet?

Reply:
Ack